### PR TITLE
chore: fix failing tests due to ff release

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Chore**

This PR resolves []

### Description

After flipping the AREnabledImprovedAlertsFlow some tests started failing. This pr fixes those tests.

### Follow - up

Update those tests to cover all new functionality

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- fix failing tests due to release of AREnabledImprovedAlertsFlow - gkartalis

</details>

### CODE REVIEW

**Comment by gkartalis** [comment]
#squashOnGreen

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (fix failing tests due to release of AREnabledImprovedAlertsFlow - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 50abb7ae283ed9b6e6b4d9d0b92183de2553887e
</p>

**Comment by MounirDhahri** [comment]
Thanks @gkartalis

File: src/lib/Components/ArtworkFilter/Filters/ArtistNationalitiesOptions.tests.tsx [hunk 1]
```diff
@@ -26,6 +27,10 @@ const MOCK_AGGREGATIONS: Aggregations = [
 ]
 
 describe("ArtistNationalitiesOptionsScreen", () => {
```
Comment (dimatretyak) [inline src/lib/Components/ArtworkFilter/Filters/ArtistNationalitiesOptions.tests.tsx:29]: **suggestion**: refactor code and use `renderWithWrappersTL`

File: src/lib/Scenes/Sale/Sale.tests.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ describe("Sale", () => {
 
   beforeEach(() => {
     mockEnvironment = createMockEnvironment()
+    __globalStoreTestUtils__?.injectFeatureFlags({ AREnableImprovedAlertsFlow: false })
```
Comment (dimatretyak) [inline src/lib/Scenes/Sale/Sale.tests.tsx:36]: we can skip `AREnableImprovedAlertsFlow` feature flag here

File: src/lib/Components/ArtworkFilter/Filters/ArtistNationalitiesOptions.tests.tsx (hunk 1 above)
Comment (gkartalis) [reply src/lib/Components/ArtworkFilter/Filters/ArtistNationalitiesOptions.tests.tsx:29]: will do that in the followup, I merged that in order to unblock beta-releases ;)

**Pull request merged**